### ISSUE TITLE
Can't access transmssion from local network

### ISSUE THREAD

**Issue by annualstatement** [haugene/docker-transmission-openvpn#486]
First, I want to thank you for an awesome container.

So, I’m running open media vault (OMV) and i am having a very hard time getting to transmissions webgui on my local network. I have tried setting LOCAL_NETWORK and have changed rpc bind addresses with no luck. However, I am able to get through via reverse proxy no problem. I have tried various combinations of LOCAL_NETWORK and rcp binds as well and nothing has worked. Every time I try to connect via local network I get the image below. I have all the forwarded ports open through my router and I am running the container in bridged mode. 

I also had to create a custom VPN provider since it wasn’t on the list. Below is my setup. Thanks in advance. 

Environmental variable used:
`#Remove # for variables you want to use
OPENVPN_PROVIDER=octanevpn
OPENVPN_CONFIG=montreal
OPENVPN_USERNAME=
OPENVPN_PASSWORD=
OPENVPN_OPTS=--inactive 3600 --ping 10 --ping-exit 60
LOCAL_NETWORK=192.168.2.1/23
#ENABLE_UFW=false
#UFW_ALLOW_GW_NET=false
#UFW_EXTRA_PORTS=
#TRANSMISSION_ALT_SPEED_DOWN=50 
#TRANSMISSION_ALT_SPEED_ENABLED=false 
#TRANSMISSION_ALT_SPEED_TIME_BEGIN=540 
#TRANSMISSION_ALT_SPEED_TIME_DAY=127 
#TRANSMISSION_ALT_SPEED_TIME_ENABLED=false 
#TRANSMISSION_ALT_SPEED_TIME_END=1020 
#TRANSMISSION_ALT_SPEED_UP=50 
#TRANSMISSION_BIND_ADDRESS_IPV4=0.0.0.0 
#TRANSMISSION_BIND_ADDRESS_IPV6=:: 
TRANSMISSION_BLOCKLIST_ENABLED=true
TRANSMISSION_BLOCKLIST_URL=http://update.transmissionbt.com/level1.gz
#TRANSMISSION_CACHE_SIZE_MB=4 
#TRANSMISSION_DHT_ENABLED=true 
TRANSMISSION_DOWNLOAD_DIR= 
#TRANSMISSION_DOWNLOAD_LIMIT=100 
#TRANSMISSION_DOWNLOAD_LIMIT_ENABLED=0 
#TRANSMISSION_DOWNLOAD_QUEUE_ENABLED=true 
TRANSMISSION_DOWNLOAD_QUEUE_SIZE=1 
#TRANSMISSION_ENCRYPTION=1 
TRANSMISSION_IDLE_SEEDING_LIMIT=0 
TRANSMISSION_IDLE_SEEDING_LIMIT_ENABLED=true 
#TRANSMISSION_INCOMPLETE_DIR=/data/incomplete 
TRANSMISSION_INCOMPLETE_DIR_ENABLED=false 
TRANSMISSION_LPD_ENABLED=true 
#TRANSMISSION_MAX_PEERS_GLOBAL=200 
#TRANSMISSION_MESSAGE_LEVEL=2 
#TRANSMISSION_PEER_CONGESTION_ALGORITHM= 
#TRANSMISSION_PEER_ID_TTL_HOURS=6 
TRANSMISSION_PEER_LIMIT_GLOBAL=70 
#TRANSMISSION_PEER_LIMIT_PER_TORRENT=50 
#TRANSMISSION_PEER_PORT=51413 
#TRANSMISSION_PEER_PORT_RANDOM_HIGH=65535 
#TRANSMISSION_PEER_PORT_RANDOM_LOW=49152 
#TRANSMISSION_PEER_PORT_RANDOM_ON_START=false 
#TRANSMISSION_PEER_SOCKET_TOS=default 
#TRANSMISSION_PEX_ENABLED=true 
#TRANSMISSION_PORT_FORWARDING_ENABLED=false 
#TRANSMISSION_PREALLOCATION=1 
#TRANSMISSION_PREFETCH_ENABLED=1 
#TRANSMISSION_QUEUE_STALLED_ENABLED=true 
#TRANSMISSION_QUEUE_STALLED_MINUTES=30 
TRANSMISSION_RATIO_LIMIT=0 
TRANSMISSION_RATIO_LIMIT_ENABLED=true 
#TRANSMISSION_RENAME_PARTIAL_FILES=true 
TRANSMISSION_RPC_AUTHENTICATION_REQUIRED=true 
TRANSMISSION_RPC_BIND_ADDRESS=0.0.0.0
#TRANSMISSION_RPC_ENABLED=true 
#TRANSMISSION_RPC_HOST_WHITELIST= 
#TRANSMISSION_RPC_HOST_WHITELIST_ENABLED=true 
TRANSMISSION_RPC_PASSWORD= 
#TRANSMISSION_RPC_PORT=9091 
#TRANSMISSION_RPC_URL=/transmission/ 
TRANSMISSION_RPC_USERNAME= 
#TRANSMISSION_RPC_WHITELIST=127.0.0.1 
#TRANSMISSION_RPC_WHITELIST_ENABLED=false 
#TRANSMISSION_SCRAPE_PAUSED_TORRENTS_ENABLED=true 
#TRANSMISSION_SCRIPT_TORRENT_DONE_ENABLED=false 
#TRANSMISSION_SCRIPT_TORRENT_DONE_FILENAME= 
#TRANSMISSION_SEED_QUEUE_ENABLED=false 
#TRANSMISSION_SEED_QUEUE_SIZE=10 
#TRANSMISSION_SPEED_LIMIT_DOWN=100 
#TRANSMISSION_SPEED_LIMIT_DOWN_ENABLED=false 
#TRANSMISSION_SPEED_LIMIT_UP=100 
#TRANSMISSION_SPEED_LIMIT_UP_ENABLED=false 
#TRANSMISSION_START_ADDED_TORRENTS=true 
#TRANSMISSION_TRASH_ORIGINAL_TORRENT_FILES=false 
#TRANSMISSION_UMASK=2 
#TRANSMISSION_UPLOAD_SLOTS_PER_TORRENT=14 
TRANSMISSION_UTP_ENABLED=false 
#TRANSMISSION_WATCH_DIR=/data/watch 
TRANSMISSION_WATCH_DIR_ENABLED=false 
#TRANSMISSION_HOME=/data/transmission-home
#WEBPROXY_ENABLED=false
#WEBPROXY_PORT=8888
`

Log:
`Using OpenVPN provider: octanevpn
Starting OpenVPN using config 
Setting OPENVPN credentials...
adding route to local network 192.168.2.1/23 via 172.17.0.1 dev eth0
RTNETLINK answers: Invalid argument
Sat Apr 21 03:29:47 2018 OpenVPN 2.4.5 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Mar  1 2018
Sat Apr 21 03:29:47 2018 library versions: OpenSSL 1.0.2g  1 Mar 2016, LZO 2.08
Sat Apr 21 03:29:47 2018 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Sat Apr 21 03:29:47 2018 TCP/UDP: Preserving recently used remote address: [AF_INET]205.204.85.246:8888
Sat Apr 21 03:29:47 2018 UDP link local: (not bound)
Sat Apr 21 03:29:47 2018 UDP link remote: [AF_INET]205.204.85.246:8888
Sat Apr 21 03:29:48 2018 [VPN] Peer Connection Initiated with [AF_INET]205.204.85.246:8888
Sat Apr 21 03:29:49 2018 TUN/TAP device tun0 opened
Sat Apr 21 03:29:49 2018 do_ifconfig, tt->did_ifconfig_ipv6_setup=0
Sat Apr 21 03:29:49 2018 /sbin/ip link set dev tun0 up mtu 1500
Sat Apr 21 03:29:49 2018 /sbin/ip addr add dev tun0 local 10.10.6.22 peer 10.10.6.21
Sat Apr 21 03:29:49 2018 /etc/openvpn/tunnelUp.sh tun0 1500 1557 10.10.6.22 10.10.6.21 init
Up script executed with tun0 1500 1557 10.10.6.22 10.10.6.21 init
Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.10.6.22
Generating transmission settings.json from env variables
sed'ing True to true

-------------------------------------
Transmission will run as
-------------------------------------
User name:   root
User uid:    0
User gid:    0
-------------------------------------

STARTING TRANSMISSION
NO PORT UPDATER FOR THIS PROVIDER
Transmission startup script complete.
Sat Apr 21 03:29:49 2018 Initialization Sequence Completed 
`
![1](https://user-images.githubusercontent.com/38576751/39079956-b1abd286-44d9-11e8-9f6a-2c35f7b9042c.png)

**Comment by haugene** [comment]
Weird. Don't know how open media vault is constructed, but is it possible that it might add another layer of networking? Is it in a VM? If docker is running on the host, and your client machine is in the 192.168.2.1/23 CIDR space, then that should work.

You should not set the bind address, there are some comments on this in #480 

And yes, the reverse proxy omits that problem. By running in the docker container network it is on the local network of the vpn container and can communicate with it without special routing rules.

**Comment by Alosander** [comment]
I seem to have the exact same problem. Also using a custom VPN provider, but not using Open Media Vault. Do tell if you find a solution.
![screenshot from 2018-04-22 20-35-48](https://user-images.githubusercontent.com/30246332/39098552-d89529f8-466c-11e8-9119-482201375219.png)

**Comment by annualstatement** [comment]
So I was trying to figure this out forever and somehow it works now. I rechecked my LOCAL_NETWORK value and realized it was wrong. I used this website http://jodies.de/ipcalc to find my correct values. Just input your NAS local ip and use the network value that is displayed. 
I initially wasn't able to connect because 1) I had the incorrect LOCAL_NETWORK and 2) I was trying to access the webgui from a remote location using VPN on top of the container running VPN, hence the error. I then used the VM on the actual host and was able to get to the webui just fine. My bad, user error.
